lib/rubocop/cop/autocorrect_logic.rb

```diff
@@ -20,7 +20,13 @@ def autocorrect_enabled?
         # allow turning off autocorrect on a cop by cop basis
         return true unless cop_config
 
-        cop_config['AutoCorrect'] != false
+        return false if cop_config['AutoCorrect'] == false
```

Comment:
I guess here some `case` might be a bit more readable then the 3 ifs. I'm with the current implementation, though.